Code/Framework/AzCore/Platform/Windows/AzCore/Debug/StackTracer_Windows.cpp

```diff
@@ -90,7 +90,7 @@ namespace AZ {
             {
             case CBA_EVENT:
                 evt = (PIMAGEHLP_CBA_EVENT)CallbackData;
-                _tprintf(_T("%s"), (PTSTR)evt->desc);
+                _tprintf(_T("%s"), evt->desc);
```

Comment:
We could switch this to regular printf.
```suggestion
                printf("%ls", evt->desc);
```

Reply:
in this case this is still using the ASCII versions, there are too many things around this file to replace ATM